### ISSUE TITLE
Overlaylayers not working

### ISSUE THREAD

**Issue by acnetj** [JeroenDeDauw/Maps#642]
### Setup

- Maps version: 7.20.1
- MW version: 1.34

### Issue

The overlaylayers parameter doesn't work. The overlaylayer should appear on top but somehow it is loaded at the bottom below the layers that are listed on the layers parameter. It only flashes a little bit when I switch between regular layers.

Example: https://www.transit.wiki/Sandbox_3

It also doesn't work here either: https://maps.extension.wiki/wiki/Leaflet_Maps_examples#Overlays

How the map should appear: https://apps.transitunlimited.org/sfwatershedmap/

**Comment by JeroenDeDauw** [comment]
Yeah now that you mention it I noticed that when creating the example on the "new" Maps wiki. It worked a year or two ago for sure.

**Comment by kghbln** [comment]
I can confirm that it is still not working. The browser console remains silent.
